suites/apis/googleServiceAccountRemovalTest.js

```diff
@@ -0,0 +1,471 @@
+const chai = require('chai');
+
+const apiUtil = require('../../utils/apiUtil.js');
+const { Bash } = require('../../utils/bash.js');
+const bash = new Bash();
+const { Commons } = require('../../utils/commons.js');
+
+
+Feature('GoogleServiceAccountRemoval');
+
+/**
+ * Test that the SA removal job, fence-create google-manage-user-registrations,
+ * deletes invalid SA and Google projects.
+ *
+ * To check that the user does not have access to the data anymore after the
+ * job cleans up a SA, we need to create a key before the clean up, and make
+ * sure we cannot use it after the clean up. Since the validation checks that
+ * there is no external access to the SA, creating a key would prevent us from
+ * testing other removal use cases. This is why the data access is only
+ * checked in the "SA has external access" test.
+ */
+
+
+BeforeSuite(async (google, fence, users) => {
+  await fence.complete.suiteCleanup(google, users);
+});
+
+
+After(async (google, fence, users) => {
+  await fence.complete.suiteCleanup(google, users);
+});
+
+
+AfterSuite(async (google, fence) => {
+  await google.unlockGoogleProject(fence.props.googleProjectDynamic);
```

Comment:
shouldn't you get the response from unlockGoogleProject and make sure it's true?